docs/general/getting-started.md

```diff
@@ -6,26 +6,28 @@ template: explore-cards.html
 
 ## What is Polkadot?
 
-Polkadot is the first layer-0 (L0) blockchain that provides shared security and secure
-interoperability to layer-1 (L1) blockchains. Those L1 blockchain attached to Polkadot are also
-called [parachains](../learn/learn-parachains.md) as their transactions are processed in parallel by
-Polkadot. For more information about Polkadot see the dedicated page about the state of
-[Polkadot 1.0](polkadot-v1.md) released in mid 2023.
+!!!info "Explore Polkadot with [secure and user-friendly wallets](https://www.polkadot.network/ecosystem/wallets/)."
 
-Polkadot has an on-chain [open governance](../learn/learn-polkadot-opengov.md) (also called Polkadot
+Polkadot is a **blockchain-powered multi-core decentralized cloud computer**. Let's break it down bit by bit:
+
+- **blockchain-powered:** Polkadot at its heart has a blockchain that permanently and immutably stores sensitive information about the protocol and applications running on top of it
+- **multi-core:** Like traditional computers, Polkadot has a virtual CPU with multiple virtual cores
+- **decentralized cloud computer:** Instead of being run by a single entity, Polkadot is run by real computers owned by different operators scattered around the globe, part of a decentralized peer-to-peer network
+
+**Polkadot is a Web3 service provider**: it provides secure and resilient computation and interoperability for any application connected to its virtual cores. Applications can be fully-fledged blockchains, known as [parachains](../learn/learn-parachains.md) or Polkadot rollups. Instead of bootstrapping their security, rollups can connect to Polkadot via its virtual cores by buying [coretime](../learn/learn-agile-coretime.md) with DOT and tap into Polkadot's multi-billion shared security. By doing so, they inherently access Polkadot's secure interoperability and can communicate with other applications running on different cores of Polkadot's virtual CPU.
```

Comment:
```suggestion
**Polkadot is a Web3 service provider**: it provides secure and resilient computation and interoperability for any application connected to its network. Applications can be fully-fledged blockchains deployed as rollups on Polkadot (also known as [parachains](../learn/learn-parachains.md)). Instead of bootstrapping their security, rollups can connect to Polkadot via its virtual cores by buying [coretime](../learn/learn-agile-coretime.md) with DOT and tap into Polkadot's multi-billion shared security. By doing so, they inherently access Polkadot's secure interoperability and can communicate with other applications running on different cores of Polkadot's virtual CPU.
```